app/controllers/cardboard/application_controller.rb

```diff
@@ -6,6 +6,10 @@ class ApplicationController < ActionController::Base
     end
     before_filter :for_gon
     protect_from_forgery
+
```

Comment:
I'm not a big fan of the placement and name of this method. How about `previews#create`? I just like RESTful stuff and creating a preview by sending it a markdown makes sense!